Library/Formula/thewidgetfactory.rb

```diff
@@ -0,0 +1,66 @@
+class Thewidgetfactory < Formula
+  desc "The Widget Factory is a showcase tool for GTK+ widgets"
+  homepage "http://www.stellingwerff.com/?page_id=10"
+  url "http://www.stellingwerff.com/TheWidgetFactory/thewidgetfactory-0.2.1.tar.gz"
+  sha256 "10bbbe50dc23a0cd47762749c6bb402559721951f3c6af40366258593bcb1ce4"
+
+  depends_on "pkg-config" => :build
+  depends_on "gtk+"
+
+  patch :DATA
```

Comment:
We're happy to do patching downstream. We're less happy to carry patches forever that upstream aren't aware of and may never accept, essentially because we end up becoming upstream at that point in keeping the software functioning as expected when upstream have stopped doing so.

It isn't really uncommon for GTK projects to throw deprecation notices, pretty much everything I use that has a GTK GUI does so.